printer_mods/hoschi/FYSETC_Spider_DIN_Mount/README.md

```diff
@@ -1,30 +1,30 @@
-FYSETC Spider DIN Rail Mount
+### FYSETC Spider DIN Rail Mount
 
 for horizontal or vertical mounting
 
 ![FYSETC_Spider_DIN_Mount](./IMG/main.jpg)
 
-Required material:
-
-2x printed DIN mounts
+Required materials:
 
 for horizontal mounting:
-spider_bracket_horizontal_terminal.stl
-spider_bracket_horizontal_usb.stl
 
-for vertical mountin:
-spider_bracket_vertical_terminal.stl
-spider_bracket_vertical_usb.stl
+- 1x printed bracket terminal side of the Spider Board: ![spider_bracket_horizontal_terminal.stl](./STL/spider_bracket_horizontal_terminal.stl)
```

Comment:
For simple links, don't put `!` in front of it - this makes it think it's an image, it's not, so it's broken.  Just

    [spider_bracket_horizontal_terminal.stl](./STL/spider_bracket_horizontal_terminal.stl)

should work.

<img width="748" alt="Screen Shot 2021-05-12 at 19 43 19" src="https://user-images.githubusercontent.com/69389593/118057595-5803ba80-b35a-11eb-963c-0768ac846a98.png">